.devcontainer/docker-compose-vscode.yml

```diff
@@ -0,0 +1,78 @@
+#  Copyright (C) 2018-2023 LEIDOS.
```

Comment:
CARMA Platform already had a [documented process](https://usdot-carma.atlassian.net/wiki/spaces/CRMPLT/pages/488505352/Setup+VS+Code+for+CARMA+Platform+Development) for using a devcontainer for development with its related repositories. Since v2x-ros-driver can be used outside of CARMA Platform, maybe it is useful to maintain its own devcontainer, but can we remove or revise this docker compose since it is requiring ros1_bridge and carma-messenger services? 

Might be worth setting up a quick 30 minute meeting with @MishkaMN and @adev4a to briefly discuss devcontainer support for our auxiliary CARMA Platform repos so that we can have a consistent approach that we can reuse in all of our repos. If it spins off more questions, maybe we hold these changes off for a separate PR/task?

Reply:
Branched to feature-dev-container, removed dev container and vscode task from this branch.